web/client/translations/data.en-US.json

```diff
@@ -762,7 +762,8 @@
             }
         },
         "locate": {
-            "tooltip": "Locate me",
+            "tooltip": "Follow my position",
```

Comment:
"Show my position" is ok. Auto follow until map pan is implicit, as in gmaps. Please fill it in other languages.

Reply:
Fixed